openmetadata-service/src/main/java/org/openmetadata/service/search/openSearch/OpenSearchClientImpl.java

```diff
@@ -789,7 +789,11 @@ private static SearchSourceBuilder addAggregation(SearchSourceBuilder builder) {
                 .size(EntityBuilderConstant.MAX_AGGREGATE_SIZE))
         .aggregation(
             AggregationBuilders.terms("entityType").field("entityType").size(EntityBuilderConstant.MAX_AGGREGATE_SIZE))
-        .aggregation(AggregationBuilders.terms("tier.tagFQN").field("tier.tagFQN"));
+        .aggregation(AggregationBuilders.terms("tier.tagFQN").field("tier.tagFQN"))
```

Comment:
@07Himank did we test this?, owner.fullyQualifiedName is marked as text https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-service/src/main/resources/elasticsearch/en/table_index_mapping.json#L296